### PR TITLE
Translate untranslated chapters to Polish

### PR DESCRIPTION
**Pull request by Unrooted**
This translates the untranslated chapter of the Tour of Scala into Polish.

### CODE REVIEW

**Comment by eed3si9n** [comment]
Thanks for the contribution! I can't review this myself so could you find someone who can? Fortunately Poland has no shortage of knowledgable Scala programmers so I'm sure you can find someone in no time. /cc @Krever, @romanowski, @rkrzewski, @EncodePanda

**Comment by Unrooted** [comment]
I added f6e4b5051b1fb8ac7d0a9a390bb92d8b40fb7fc4 to the #1758 a few days ago and it's also added in here, I guess we can close #1758 and focus on this PR since it also does include the same thing but doesn't have such a mess in logs.

**Review by Krever — CHANGES_REQUESTED** [review]
Few changes, but looks good in general, thanks! 
I haven't checked the new pages ordering though, I hope its alright.

**Review by Krever — APPROVED** [review]
LGTM. Would be good if one more person could look at it, but if not should still be ok.

File: _pl/tour/by-name-parameters.md [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+title: Parametry przekazywane według nazwy
 partof: scala-tour
+
+num: 32
+next-page: annotations
+previous-page: operators
 language: pl
 ---
+
+_Parametry przekazywane według nazwy_ są ewaluowane za każdym razem gdy są używane. Nie zostaną w ogóle wyewaluowane jeśli nie będą używane. Jest to podobne do zastępowania parametrów według nazwy przekazanymi wyrażeniami. Są przeciwieństwem do _parametrów według wartości_. Aby utworzyć parametr wywoływany według nazwy, po prostu dodaj `=>` przed jego typem.
```
Comment (rkrzewski) [inline _pl/tour/by-name-parameters.md:12]:
```suggestion
_Parametry przekazywane według nazwy_ są ewaluowane za każdym razem gdy są używane. Nie zostaną w ogóle wyewaluowane jeśli nie będą używane. Jest to podobne do zastępowania parametrów w ciele funkcji wyrażeniami podanymi w miejscu jej wywołania. Są przeciwieństwem do _parametrów przekazywanych według wartości_. Aby utworzyć parametr przekazywany według nazwy, po prostu dodaj `=>` przed jego typem.
```

File: _pl/tour/by-name-parameters.md [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+
+whileLoop (i > 0) {
+  println(i)
+  i -= 1
+}  // prints 2 1
+```
+
+Metoda `whileLoop` używa wielu list parametrów do określenia warunku i treści pętli. Jeśli `condition` (warunek) jest prawdziwy, `body` (treść) jest wykonywana a następnie wykonywane jest rekurencyjne wywołanie `whileLoop`. Jeśli `condition` jest fałszywy, treść nigdy nie jest ewaluowana, ponieważ dodaliśmy `=>` do typu `body`.
+
+Teraz, kiedy przekażemy `i>0` jako nasz `condition` (warunek) i `println(i); i-= 1` jako `body`, nasze wyrażenie zachowuje się jak standardowa pętla while w wielu językach.
```
Comment (rkrzewski) [inline _pl/tour/by-name-parameters.md:39]:
```suggestion
Jeśli przekażemy `i>0` jako nasz `condition` (warunek) i `println(i); i-= 1` jako `body`, nasze wyrażenie zachowuje się jak standardowa pętla while w wielu językach.
```

File: _pl/tour/by-name-parameters.md [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+
+Ta możliwość opóźnienia ewaluacji parametru do czasu jego użycia może zwiększyć wydajność, jeśli ewaluacja parametru wymaga intensywnych obliczeń lub dłużej działającego bloku kodu, takiego jak pobieranie adresu URL.
```
Comment (rkrzewski) [inline _pl/tour/by-name-parameters.md:41]:
```suggestion
Ta możliwość opóźnienia ewaluacji parametru do czasu jego użycia może zwiększyć wydajność, jeśli ewaluacja parametru wymaga intensywnych obliczeń lub dłużej działającego bloku kodu, takiego jak pobieranie treści spod adresu URL.
```

File: _pl/tour/package-objects.md [hunk 4]
```diff
@@ -1,14 +1,68 @@
...
 previous-page: packages-and-imports
+language: pl
 ---
 
-# Package objects
+# Obiekty pakietu
+
+Scala udostępnia obiekty pakietu jako wygodny kontener wspóldzielony w całym pakiecie.
+
+Obiekty pakietu mogą zawierać dowolne definicje, nie tylko definicje zmiennych i metod. Na przykład, są często używane do przechowywania aliasów typów dla całego pakietu i niejawnych konwersji. Obiekty pakietu mogą nawet dziedziczyć klasy i cechy (traits) Scali.
```
Comment (rkrzewski) [inline _pl/tour/package-objects.md:16]:
```suggestion
Obiekty pakietu mogą zawierać dowolne definicje, nie tylko definicje zmiennych i metod. Na przykład, są często używane do przechowywania aliasów typów i niejawnych konwersji. Obiekty pakietu mogą nawet dziedziczyć klasy i cechy (traits) Scali.
```

File: _pl/tour/package-objects.md [hunk 5]
```diff
@@ -1,14 +1,68 @@
...
+
+Z założenia, kod źródłowy obiektu pakietu jest zwykle umieszczany w pliku źródłowym o nazwie `package.scala`.
```
Comment (rkrzewski) [inline _pl/tour/package-objects.md:18]:
```suggestion
Zgodnie z konwencją, kod źródłowy obiektu pakietu jest zwykle umieszczany w pliku źródłowym o nazwie `package.scala`.
```

File: _pl/tour/package-objects.md [hunk 6]
```diff
@@ -1,14 +1,68 @@
...
+package gardening.fruits
+
+case class Fruit(name: String, color: String)
+object Apple extends Fruit("Apple", "green")
+object Plum extends Fruit("Plum", "blue")
+object Banana extends Fruit("Banana", "yellow")
+```
+
+Teraz załóżmy, że chcesz umieścić zmienną `planted` i metodę `showFruit` bezpośrednio w pakiecie `gardening.fruits`.
+Oto jak to się robi:
```
Comment (rkrzewski) [inline _pl/tour/package-objects.md:35]:
```suggestion
Możesz zrobić to w następujący sposób:
```

File: _pl/tour/package-objects.md [hunk 7]
```diff
@@ -1,14 +1,68 @@
...
+package gardening
+package object fruits {
+  val planted = List(Apple, Plum, Banana)
+  def showFruit(fruit: Fruit): Unit = {
+    println(s"${fruit.name}s are ${fruit.color}")
+  }
+}
+```
+
+Jako przykład tego, jak wygląda użytkowanie, obiekt `PrintPlanted` importuje `planted` i `showFruit` w ten sam sposób, w jaki importuje klasę `Fruit` - używając importu wieloznacznego w pakiecie `gardening.fruits`.
```
Comment (rkrzewski) [inline _pl/tour/package-objects.md:48]:
```suggestion
Jako przykład tego, jak wygląda użycie definicji przygotowanych w ten sposób, obiekt `PrintPlanted` importuje `planted` i `showFruit` w ten sam sposób, w jaki importuje klasę `Fruit` - używając importu wieloznacznego w pakiecie `gardening.fruits`.
```

File: _pl/tour/packages-and-imports.md [hunk 8]
```diff
@@ -1,15 +1,92 @@
...
+package com.google.selfdrivingcar.camera
+
+class Lens
+```
+
+Może to odpowiadać następującej strukturze katalogów: `SelfDrivingCar/src/main/scala/com/google/selfdrivingcar/camera/Lens.scala`.
+
+## Import
+
+Deklaracje `import` służą do uzyskiwania dostępu do członków (members, tzn.: klasy, cechy, funkcje itp.) w innych pakietach. Aby uzyskać dostęp do członków tego samego pakietu, nie jest wymagana deklaracja `import`. Deklaracje `import` są selektywne.
```
Comment (rkrzewski) [inline _pl/tour/packages-and-imports.md:66]:
```suggestion
Deklaracje `import` służą do uzyskiwania dostępu do elementów składowych (members, tzn.: klasy, cechy, funkcje itp.) w innych pakietach. Aby uzyskać dostęp do elementów tego samego pakietu, nie jest wymagana deklaracja `import`. Deklaracje `import` są selektywne.
```

File: _pl/tour/packages-and-imports.md [hunk 9]
```diff
@@ -1,15 +1,92 @@
...
+
+```
+import users._  // zaimportuj wszystko z pakietu użytkowników
+import users.User  // zaimportuj klasę User
+import users.{User, UserPreferences}  // zaimportuj tylko wybranych członków
```
Comment (rkrzewski) [inline _pl/tour/packages-and-imports.md:71]:
```suggestion
import users.{User, UserPreferences}  // zaimportuj tylko wybrane elementy
```

File: _pl/tour/packages-and-imports.md [hunk 10]
```diff
@@ -1,15 +1,92 @@
...
+import users.{UserPreferences => UPrefs}  // zaimportuj i zmień nazwę dla wygody
+```
+
+Jedną z różnic w Scali od Javy jest to, że `import` można używać wszędzie:
```
Comment (rkrzewski) [inline _pl/tour/packages-and-imports.md:75]:
```suggestion
Jedną z różnic w Scali od Javy jest to, że deklarację `import` można umieścić w dowolnym miejscu:
```

File: _pl/tour/packages-and-imports.md [hunk 11]
```diff
@@ -1,15 +1,92 @@
...
+
+```tut
+def sqrtplus1(x: Int) = {
+  import scala.math.sqrt
+  sqrt(x) + 1.0
+}
+```
+
+W przypadku konfliktu nazw i konieczności zaimportowania czegoś z katalogu głównego projektu, poprzedź nazwę pakietu przedrostkiem `_root_`:
```
Comment (rkrzewski) [inline _pl/tour/packages-and-imports.md:84]:
```suggestion
W przypadku konfliktu nazw i konieczności podania pełnej ścieżki w hierarchii nazw pakietów, poprzedź nazwę pakietu przedrostkiem `_root_`:
```

File: _pl/cheatsheets/index.md [hunk 12]
```diff
@@ -1,88 +1,627 @@
...
-|  `class C extends D { override def f = ...}`	                                                           |  w przeciążeniach funkcji wymagane jest słowo kluczowe override. |
-|  `new java.io.File("f")`                   	                                                           |  tworzenie obiektu. |
-|  <span class="label important">Źle</span> `new List[Int]`<br> <span class="label success">Dobrze</span> `List(1,2,3)` |  błąd typu: typ abstrakcyjny<br>zamiast tego konwencja: wywoływalna fabryka przysłaniająca typ |
-|  `classOf[String]`                                                                                       |  literał klasy. |
-|  `x.isInstanceOf[String]`                                                                                |  sprawdzenie typu (w czasie wykonania) |
-|  `x.asInstanceOf[String]`                                                                                |  rzutowanie typu (w czasie wykonania) |
-|  `x: String`                                                                                             |  oznaczenie typu (w czasie kompilacji) |
+<table>
```
[the remark this answers is not in the document]
Comment (rkrzewski) [reply _pl/cheatsheets/index.md:17]: Markdown code was replaced by HTML here. Was that intended?

**Review by rkrzewski — COMMENTED** [review]
Hey, I've added some edit suggestions for the consideration of translation's author.

**Comment by Unrooted** [comment]
Yes, it was intended. I did it to make it consistent with the English version of the cheat sheet.

**Comment by SethTisue** [comment]
Is this ready for merge?

**Comment by Unrooted** [comment]
Yes, it is. I added all requested changes, it is ready for merge.

**Comment by SethTisue** [comment]
thank you, @Unrooted and reviewers, for this substantial contribution!

**Pull request merged**